[dynamo] add error message for unsupported LOAD_BUILD_CLASS

williamwen42 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #150341
* __->__ #150323

Improved error message for https://github.com/pytorch/pytorch/issues/128942

cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @chenyang78 @kadeng @chauhang @amjames

williamwen42 committed:
[dynamo] add error message for unsupported LOAD_BUILD_CLASS

[ghstack-poisoned]

zou3519 reviewed (commented):
message looks good to me but your tests are failing

williamwen42 committed:
Update on "[dynamo] add error message for unsupported LOAD_BUILD_CLASS"


Improved error message for https://github.com/pytorch/pytorch/issues/128942

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 kadeng chauhang amjames

[ghstack-poisoned]

williamwen42 committed:
Update on "[dynamo] add error message for unsupported LOAD_BUILD_CLASS"


Improved error message for https://github.com/pytorch/pytorch/issues/128942

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 kadeng chauhang amjames

[ghstack-poisoned]

williamwen42 commented:
@pytorchbot merge -i
